webknossos/tests/test_dataset.py

```diff
@@ -1050,13 +1050,13 @@ def test_writing_subset_of_compressed_data() -> None:
                 data=(np.random.rand(10, 10, 10) * 255).astype(np.uint8),
             )
 
-        assert compressed_mag._mag_view_bounding_box_at_creation == BoundingBox(
+        assert compressed_mag.bounding_box == BoundingBox(
             topleft=(
                 0,
                 0,
                 0,
             ),
-            size=(120, 140, 160),
```

Comment:
`_mag_view_bounding_box_at_creation` used to be in the current mag. Instead `bounding_box` is now used for all checks and is handled in mag1.